### ISSUE TITLE
Incorrect version number from cli?

### ISSUE THREAD

**Issue by ManzDev** [postcss/postcss-cli#405]
With postcss-cli 8.3.1 works fine:

```bash
$ npm install -D postcss-cli@8.3.1
$ npx postcss --version
8.3.1
```

With postcss-cli 9.0.0+ don't works:

```bash
$ npm install -D postcss-cli@9.0.1
$ npx postcss --version
1.0.0
```

**Comment by RyanZim** [comment]
Yep, seems to be a known bug with our argument parser in ESM modules: https://github.com/yargs/yargs/issues/1934

**Comment by RyanZim** [comment]
PR: https://github.com/postcss/postcss-cli/pull/406

**Comment by lk77** [comment]
Yeap i have the same issue it shows 0.0.0

**Comment by RyanZim** [comment]
Yes, this is a known bug with a dependency we use; if anyone can work on fixing https://github.com/yargs/yargs/issues/1934, it would be appreciated.

**Comment by franz-josef-kaiser** [comment]
Releases also lag behind tags somehow.

**Comment by RyanZim** [comment]
@franz-josef-kaiser https://github.com/postcss/postcss-cli/blob/master/CHANGELOG.md has the updated info; I've stopped updating GH releases, as it's just duplicate work.

**Comment by RyanZim** [comment]
Seems to be working on the latest version:
```
$ postcss --version
11.0.1
```
